packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -927,13 +929,19 @@ export class ProjectService extends BaseService {
     }
 
     async delete(projectUuid: string, user: SessionUser): Promise<void> {
-        const { organizationUuid, type } =
-            await this.projectModel.getWithSensitiveFields(projectUuid);
+        const project = await this.projectModel.getWithSensitiveFields(
+            projectUuid,
+        );
 
         if (
             user.ability.cannot(
                 'delete',
-                subject('Project', { organizationUuid, projectUuid }),
+                subject('Project', {
+                    type: project.type,
+                    projectUui: project.projectUuid,
```

Comment:
```suggestion
                    projectUuid: project.projectUuid,
```

Reply:
These typos are very annoying, if you don't type it corrently, then the ability check is useless. We should enforce this typing somehow.